.github/workflows/deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Test
+    branches:
+      - main
+      - development
+    types:
+      - completed
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: ${{ github.event.workflow_run.head_branch || github.ref }}
+
+      - name: Set BRANCH_TAG
+        uses: nimblehq/branch-tag-action@v1.2
+        with:
+          ref: ${{ github.event.workflow_run.head_branch || github.ref }}
+
+      - name: Set HEROKU_APP_NAME
+        run: |
+          if [[ $BRANCH_TAG = "latest" ]]
+          then
+            echo "HEROKU_APP_NAME=${{ secrets.HEROKU_APP_NAME }}" >> $GITHUB_ENV
+          else
+            echo "HEROKU_APP_NAME=${{ secrets.HEROKU_APP_NAME }}-staging" >> $GITHUB_ENV
+          fi
+
+      - uses: akhileshns/heroku-deploy@v3.7.8
+        with:
+          heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
+          heroku_app_name: $HEROKU_APP_NAME
+          heroku_email: ${{ secrets.HEROKU_APP_NAME }}
```

Comment:
Just curious, why did you use `HEROKU_APP_NAME` for email?

Reply:
@hoangmirs My bad 🙌
I copy the wrong one. Actually I have `HEROKU_APP_EMAIL` for that part.

<img width="247" alt="Screen Shot 2563-12-23 at 16 55 58" src="https://user-images.githubusercontent.com/749672/102984207-d395d400-453f-11eb-90cd-f20c124bd797.png">